### ISSUE TITLE
[Update Request]: Avidemux.Avidemux 2.8.1

### ISSUE THREAD

**Issue by luzpaz** [microsoft/winget-pkgs#193156]
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

Avidemux.Avidemux

### Package Version

2.8.1

### Please describe the changes you would like to see

https://github.com/mean00/avidemux2/releases/tag/2.8.1

**Comment by Dragon1573** [comment]
> [!WARNING]
>
> Close with reason: https://github.com/microsoft/winget-pkgs/labels/Blocking-Issue https://github.com/microsoft/winget-pkgs/labels/Interactive-Only-Installer
>
> - See https://github.com/microsoft/winget-pkgs/issues/148012#issuecomment-2041286415
> - Previous failed attempt, see #167336

```text
01:47:45 D:\...\winget-pkgs  [master ≡ +1 ~0 -0 !] 14ms pwsh> C:\Users\Dragon1573\Downloads\Avidemux.Avidemux_2.8.1\Avidemux_2.8.1_X64_nullsoft_en-US.exe --version
IFW Version: 3.0.6, built with Qt 5.9.5.
Build date: Dec 11 2018
Installer Framework SHA1: 19b0bb9
```

In previous version `v2.8.0.220109`, a Nullsoft installer wizard `avidemux_2.8.0_win64.exe` is provided. But in this version, the Nullsoft installer wizard is missing.

```text
01:53:12 D:\...\winget-pkgs  [master ≡ +1 ~0 -0 !] 1ms pwsh> komac show --installer-manifest Avidemux.Avidemux            
PackageIdentifier: Avidemux.Avidemux
PackageVersion: 2.8.0.220109
Platform:
- Windows.Desktop
InstallerType: nullsoft
InstallModes:
- interactive
- silent
InstallerSuccessCodes:
- 1223
UpgradeBehavior: install
ReleaseDate: 2022-01-10
Installers:
- Architecture: x64
  InstallerUrl: https://github.com/mean00/avidemux2/releases/download/2.8.0/avidemux_2.8.0_win64.exe
  InstallerSha256: 5B3AEABEAAD29B785C76C87536054AD3C98618037E9E43812543CB555405E894
  ProductCode: Avidemux 2.8 - 64 bits (64-bit)
ManifestType: installer
ManifestVersion: 1.4.0
```

**Comment by luzpaz** [comment]
Hi @eumagga0x2a, can you weigh-in on this ?

**Comment by luzpaz** [comment]
Hi @mean00, would it be possible for you to weigh-in on this ?

**Comment by mean00** [comment]
in case it helps, the visual c++ builds are packaged using qt installer
( https://doc.qt.io/qtinstallerframework/ )

**Comment by luzpaz** [comment]
Does winget support qtinstaller installations ?

**Comment by Dragon1573** [comment]
> Does winget support qtinstaller installations ?

Yes, but seems a specific series of versions. That exact version of QTIFW shown in above comment is not supported by WinGet. It seems we need QTIFW v4.

The Qt Framework and the related Install Framework might requires update.

**Comment by denelon** [comment]
This version (or a newer version) of Avidemux.Avidemux (2.8.1) has been added to winget-pkgs via PR #352924. Closing as resolved.
